app/views/spree/admin/orders/_form/_adjustments.html.haml

```diff
@@ -0,0 +1,14 @@
+- if adjustments.present?
+  %fieldset.no-border-bottom
+    %legend= title
+    %table>
+      %thead
+        %th= Spree.t('name')
+        %th= Spree.t('amount')
+      %tbody#order-charges.with-border
+        - adjustments.each do |adjustment|
+          - if (adjustment.originator_type != 'Spree::ShippingMethod') && !(adjustment.originator_type == 'Spree::TaxRate' && adjustment.amount == 0)
```

Comment:
Along the same lines, this couples a lot this view to the inner details of the adjustment. If we don't want to render anything for these kinds of adjustments we can avoid rendering the table altogether right?

We could accomplish this by loading from DB or filtering only the adjustments we care about and passing those in the `adjustments` local. This would make things a lot more maintainable IMO.

Reply:
yes, I agree, I noticed that problem. this is a problem in the spree code.
my decision was not to deviate too much from the spree code...

do you think I should do some magic in _form to get only adjustments that pass the conditional?
to be honest, instead, I think we should aim to remove these conditionals, these views should just show what's in the DB...

Reply:
> do you think I should do some magic in _form to get only adjustments that pass the conditional?
to be honest, instead, I think we should aim to remove these conditionals, these views should just show what's in the DB... 

I would just load from DB those that we care about in this partial (shipping method nad tax rate) or do you mean we should display all kinds of adjustments?

Reply:
I created https://github.com/openfoodfoundation/openfoodnetwork/issues/3476 for this

Reply:
ok, thank for creating the issue. I think this can definitely be improved!

Reply:
yes, I meant that we should display all kinds of adjustments.